.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
             "url": "/.vscode/schema/locale.json"
         }
     ],
-    "cSpell.words": ["Reificated"]
+    "cSpell.words": ["Reificated"],
+    "i18n-ally.localesPaths": "packages/maskbook/src/plugins/Trader/messages"
```

Comment:
revert this change plz

Reply:
This is automatically modified by vscode

Reply:
this is modified by prettier

Reply:
no it's not, it's added by i18n-ally vscode plugin

Reply:
Yes!